src/utils.cpp

```diff
@@ -281,125 +281,124 @@ std::string getExpandedLocationCode(const std::string& unexpandedLocationCode,
     {
-        try
-        {
-            // Expanded location code is formaed by combining two keywords
-            // depending on type in unexpanded one. Second one is always "SE".
-            std::string kwd1, kwd2{"SE"};
+        // Expanded location code is formaed by combining two keywords
+        // depending on type in unexpanded one. Second one is always "SE".
+        std::string kwd1, kwd2{"SE"};
 
-            // interface to search for required keywords;
-            std::string kwdInterface;
+        // interface to search for required keywords;
+        std::string kwdInterface;
 
-            // record which holds the required keywords.
-            std::string recordName;
+        // record which holds the required keywords.
+        std::string recordName;
 
-            auto pos = unexpandedLocationCode.find("fcs");
+        auto pos = unexpandedLocationCode.find("fcs");
+        if (pos != std::string::npos)
+        {
+            kwd1 = "FC";
+            kwdInterface = "com.ibm.ipzvpd.VCEN";
+            recordName = "VCEN";
+        }
+        else
+        {
+            pos = unexpandedLocationCode.find("mts");
             if (pos != std::string::npos)
             {
-                kwd1 = "FC";
-                kwdInterface = "com.ibm.ipzvpd.VCEN";
-                recordName = "VCEN";
+                kwd1 = "TM";
+                kwdInterface = "com.ibm.ipzvpd.VSYS";
+                recordName = "VSYS";
             }
             else
             {
-                pos = unexpandedLocationCode.find("mts");
-                if (pos != std::string::npos)
-                {
-                    kwd1 = "TM";
-                    kwdInterface = "com.ibm.ipzvpd.VSYS";
-                    recordName = "VSYS";
-                }
-                else
-                {
-                    throw std::runtime_error(
-                        "Error detecting type of unexpanded location code.");
-                }
-            }
-
-            std::string firstKwdValue, secondKwdValue;
-            auto itrToVCEN = (*ipzVpdMap).find(recordName);
-            if (itrToVCEN != (*ipzVpdMap).end())
-            {
-                // The exceptions will be cautght at end.
-                getKwVal(itrToVCEN->second, kwd1, firstKwdValue);
-                getKwVal(itrToVCEN->second, kwd2, secondKwdValue);
+                throw std::runtime_error(
+                    "Error detecting type of unexpanded location code.");
             }
-            else
-            {
-                std::array<const char*, 1> interfaceList = {
-                    kwdInterface.c_str()};
+        }
 
-                types::MapperGetObject mapperRetValue =
-                    getObjectMap("/xyz/openbmc_project/inventory/system/"
-                                 "chassis/motherboard",
-                                 interfaceList);
+        std::string firstKwdValue, secondKwdValue;
+        types::IPZVpdMap::const_iterator itrToVCEN;
```

Comment:
I have  removed top if case and only added this if case for getExpandedLocationCode API.
all these changes appearing because of the clang alignment happened.

Reply:
Why this change was required, please mention that as well.

Reply:
I initially thought getting IPZVpdMap type from an empty VPDMapVariant and dereferencing an empty map will cause an issue, so bought this check inside to get location code from DBus in case priming workflow.
I have checked the code again and find out that this change is not required.
I will revert back this change.

Reply:
Ok please update along with another comments.